src/apps/answers/tasks.py

```diff
@@ -75,3 +79,29 @@ async def create_report(
     finally:
         if not isinstance(session_maker, AsyncSession):
             await session_maker.remove()
+
+
+@broker.task(
+    task_name="apps.answers.tasks:removing_outdated_answers",
+    schedule=[{"cron": "*/30 * * * *"}],
+)
+async def removing_outdated_answers():
+    session_maker = session_manager.get_session()
+    try:
+        async with session_maker() as session:
+            applets_data: Result = await AppletsCRUD(
+                session
+            ).get_every_non_indefinitely_applet_retentions()
+            for applet_data in applets_data:
+                applet_id, retention_period, retention_type = applet_data
+                retention_type = DataRetention(retention_type)
+                await AnswersCRUD(session).removing_outdated_answers(
```

Comment:
Answers can be stored on arbitrary server. Please check how it works and change the code accordingly

Reply:
It fixed.